CHANGES_UNRELEASED.md

```diff
@@ -37,7 +37,9 @@
 - iOS: Added `FxAccountManager.getSessionToken`. Note that you should request the `.session` scope in the constructor for this to work properly. ([#2638](https://github.com/mozilla/application-services/pull/2638))
 - iOS: Added `FxAccountManager.getManageAccountURL`. ([#2658](https://github.com/mozilla/application-services/pull/2658))
 - iOS: Added `FxAccountManager.getTokenServerEndpointURL`. ([#2658](https://github.com/mozilla/application-services/pull/2658))
-
+- iOS: Added migration methods to `FxaAccountManager` ([#2637](https://github.com/mozilla/application-services/pull/2637)):
+  - `migrationAuthentication` will try to authenticate an account without any user intervention using previously stored account information.
```

Comment:
nit: would "user interaction" be more appropriate here than "user intervention" (which might connote the user intervening to prevent something bad from happening)?